core/DataAccess/Model.php

```diff
@@ -786,10 +788,35 @@ public function isArchiveAlreadyInProgress($invalidatedArchive)
             $bind[] = $invalidatedArchive['report'];
         }
 
-        $sql = "SELECT MAX(idinvalidation) FROM `$table` WHERE idsite = ? AND date1 = ? AND date2 = ? AND `period` = ? AND `name` = ? AND status = 1 AND $reportClause";
+        $sql = "SELECT MAX(idinvalidation) as idinvalidation, ts_invalidated FROM `$table` WHERE idsite = ? AND date1 = ? AND date2 = ? AND `period` = ? AND `name` = ? AND status = 1 AND $reportClause";
```

Comment:
not sure it's possible or make sense but do we also need a max or min on the `ts_invalidated`?

Reply:
This was a mistake I fixed locally, should be a LIMIT 1 query.

Reply:
👍